### PR TITLE
Feature/feedback form

### PR DESCRIPTION
**Pull request by vinithanatarajan**
Formtastic styles were only removed from admin UI and not from the feedback form this was causing an error in the contact us page where the feedback form lives. This PR fixes the error in the feedback form.

### CODE REVIEW

File: app/views/pages/feedback.html.haml [hunk 1]
```diff
@@ -3,27 +3,74 @@
...
+            = item
+      .small-12
+        %label
+          = f.label :nzsl_level, "Are you a:"
+          - ["Fluent NZSL user", "NZSL user", "NZSL learner"].each do |category|
+            = f.radio_button :nzsl_level, category, class: "item-style"
+            = category
+      .small-12
+        %label
+          %input{ type: "checkbox", id: "feedback_include_sign", class: "feedback-select-style" }
```
Comment (Br3nda) [inline app/views/pages/feedback.html.haml]: `%input.feedback-select-style` will make hound happy

Comment (vinithanatarajan) [inline app/views/pages/feedback.html.haml]: 👍

File: app/assets/stylesheets/application.scss [hunk 2]
```diff
@@ -329,8 +329,19 @@ abbr {
 .field_with_errors {
   display: inline;
   .text-input { border-color: $required-red; }
+  .message-input { border-color: $required-red; }
 }
 
+.item-style { margin: 8px; }
+
+.feedback-select-style { margin: 7px 7px 7px 0; }
```
Comment (Br3nda) [inline app/assets/stylesheets/application.scss]: i don't know what this means -- super interested in what you find in google.

File: .travis.yml [hunk 3]
```diff
@@ -17,4 +17,6 @@ script:
   - brakeman --run-all-checks --exit-on-warn --format plain .
   - bundle exec rspec spec
 after_success:
-  - ./cc-test-reporter after-build --id $CC_TEST_REPORTER_ID
+  - ./cc-test-reporter after-build --exit-code $TRAVIS_TEST_RESULT
```
Comment (Br3nda) [inline .travis.yml]: does it not need the reporter id to send to travis? how does it work?

File: .travis.yml [hunk 4]
```diff
@@ -1,20 +1,24 @@
...
-  - curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
-  - chmod +x ./cc-test-reporter
-  - RAILS_ENV=test bundle exec rake db:create db:schema:load
+- gem install  --no-rdoc --no-ri 'brakeman' 'bundler-audit'
+- curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
+- chmod +x ./cc-test-reporter
+- RAILS_ENV=test bundle exec rake db:create db:schema:load
+addons:
+  code_climate:
+    repo_token: 50908698e3f680091760f1b1d09e447736d9eb973a44ce91bd3ff6aa7a8c929e
```
Comment (Br3nda) [inline .travis.yml]: Does this need to be here? There's `$CODECLIMATE_REPO_TOKEN` already set in the env on travis.
Does it need it set in both places?

**Comment by vinithanatarajan** [comment]
@Br3nda I have removed the repo token.

**Pull request merged**